spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/servlet/ServletManagementChildContextConfiguration.java

```diff
@@ -147,12 +149,12 @@ abstract static class AccessLogCustomizer implements Ordered {
 
 		private static final String MANAGEMENT_PREFIX = "management_";
 
-		protected String customizePrefix(String prefix) {
+		protected String customizePrefix(String prefix, String accessLogPrefix) {
```

Comment:
Instead of changing the signature and storing the `ManagementServerProperties` in every webserver implementation, you could add a `private final ManagementServerProperties managementServerProperties` to `AccessLogCustomizer` and use the value in `customizePrefix`.

Reply:
`AccessLogCustomizer` is abstract, if I add a constructor to it, I will need to call `super(ManagementServerProperties)` in every implementation anyway. Is this what you want ?

Reply:
Yes.

Reply:
done